# HG changeset patch
# User Michael Goffioul <michael.goffioul@gmail.com>
# Date 1356232593 0
#      Sun Dec 23 03:16:33 2012 +0000
# Node ID fe2cefaa772b70b866f00e9dec95a3c7e263c389
# Parent  48a75e6884ecc7ad04329e1b9c2dfa262f981d16
Replace JAVAPKG_API with OCTINTERP_API

* ov-java.h: Replace all occurrences of JAVAPKG_API with OCTINTERP_API.

diff --git a/libinterp/octave-value/ov-java.h b/libinterp/octave-value/ov-java.h
--- a/libinterp/octave-value/ov-java.h
+++ b/libinterp/octave-value/ov-java.h
@@ -23,22 +23,16 @@ along with Octave; see the file COPYING.
 #if !defined (octave_java_h)
 #define octave_java_h 1
 
 #include <jni.h>
 
 #include <oct-obj.h>
 #include <ov.h>
 
-#ifdef JAVAPKG_BUILD
-# define JAVAPKG_API OCTAVE_EXPORT
-#else
-# define JAVAPKG_API OCTAVE_IMPORT
-#endif
-
 template <class T>
 class java_local_ref
 {
 public:
 
   java_local_ref (JNIEnv *_env)
     : jobj (0), detached (false), env (_env) { }
 
@@ -84,43 +78,43 @@ typedef java_local_ref<jobject> jobject_
 typedef java_local_ref<jclass> jclass_ref;
 typedef java_local_ref<jstring> jstring_ref;
 typedef java_local_ref<jobjectArray> jobjectArray_ref;
 typedef java_local_ref<jintArray> jintArray_ref;
 typedef java_local_ref<jbyteArray> jbyteArray_ref;
 typedef java_local_ref<jdoubleArray> jdoubleArray_ref;
 typedef java_local_ref<jthrowable> jthrowable_ref;
 
-extern JAVAPKG_API std::string
+extern OCTINTERP_API std::string
 jstring_to_string (JNIEnv* jni_env, jstring s);
 
-extern JAVAPKG_API std::string
+extern OCTINTERP_API std::string
 jstring_to_string (JNIEnv* jni_env, jobject obj);
 
-extern JAVAPKG_API octave_value
+extern OCTINTERP_API octave_value
 box (JNIEnv* jni_env, jobject jobj, jclass jcls = 0);
 
-extern JAVAPKG_API octave_value
+extern OCTINTERP_API octave_value
 box_more (JNIEnv* jni_env, jobject jobj, jclass jcls = 0);
 
-extern JAVAPKG_API int
+extern OCTINTERP_API int
 unbox (JNIEnv* jni_env, const octave_value& val, jobject_ref& jobj,
        jclass_ref& jcls);
 
-extern JAVAPKG_API int
+extern OCTINTERP_API int
 unbox (JNIEnv* jni_env, const octave_value_list& args,
        jobjectArray_ref& jobjs, jobjectArray_ref& jclss);
 
-extern JAVAPKG_API bool Vjava_matrix_autoconversion;
+extern OCTINTERP_API bool Vjava_matrix_autoconversion;
 
-extern JAVAPKG_API bool Vjava_unsigned_autoconversion;
+extern OCTINTERP_API bool Vjava_unsigned_autoconversion;
 
-extern JAVAPKG_API bool Vdebug_java;
+extern OCTINTERP_API bool Vdebug_java;
 
-class JAVAPKG_API octave_java : public octave_base_value
+class OCTINTERP_API octave_java : public octave_base_value
 {
 public:
 
   octave_java (void)
     : octave_base_value (), java_object (0), java_class (0)
   { }
 
   octave_java (const octave_java& jobj)
